src/ProgressOnderwijsUtils/MetaObject/MetaObjectPropertyLoaderAttribute.cs

```diff
@@ -1,7 +1,13 @@
 ﻿using System;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
 
 // ReSharper disable once CheckNamespace
 namespace ProgressOnderwijsUtils
 {
     public sealed class MetaObjectPropertyLoaderAttribute : Attribute, IPropertiesAreUsedImplicitly { }
+
+    public interface IConverterSource<TModel, TProvider>
+    {
+        ValueConverter<TModel, TProvider> GetValueConverter();
+    }

```

Comment:
Deze lijkt wel erg op https://github.com/aspnet/EntityFrameworkCore/blob/master/src/EFCore/ValueGeneration/IValueGeneratorSelector.cs, voor wat het waard is.